framework/src/main/java/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -209,28 +209,38 @@ public class BaseTypeVisitor<Factory extends GenericAnnotatedTypeFactory<?, ?, ?
   /** The @{@link Deterministic} annotation. */
   protected final AnnotationMirror DETERMINISTIC =
       AnnotationBuilder.fromClass(elements, Deterministic.class);
+
```

Comment:
Undoing the whitespace changes might make merging easier later on, if it's quick to do.

Reply:
I got errors without this. Some part of the build complains. I'll try again.

Reply:
I guess I can use the `--no-very` flag.

Reply:
It worked!